comm/p2p/libp2p.go

```diff
@@ -191,7 +191,7 @@ func (c Libp2pCommunication) sendMessage(
 
 	err = WriteStream(msg, bufio.NewWriterSize(stream, defaultBufferSize))
 	if err != nil {
-		c.logger.Error().Str("To", string(to)).Err(err).Msg("unable to send message")
+		c.logger.Error().Str("To", to.String()).Err(err).Msg("unable to send message")
```

Comment:
I think it should be Pretty(). 
String is formatted weird.

Reply:
But pretty is Deprecated
```
// Pretty returns a base58-encoded string representation of the ID.
// Deprecated: use String() instead.
```

Reply:
```
func (id ID) Pretty() string {
	return id.String()
}
```

Reply:
Oh, I guess it is fine then